sources/pmwebapi_test.go

```diff
@@ -23,6 +23,13 @@ type AutoGenerated struct {
 	URL    string `json:"url"`
 }
 
+type testlables struct {
+	Units    string `json:"units"`
```

Comment:
Technically, the `testlables` struct isn't JSON, but just a string, so it would probably be best to get rid of the `json:"units"` part. Though it's not a huge deal.